<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1300</width>
    <height>775</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>1151</width>
     <height>701</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="sizeConstraint">
     <enum>QLayout::SetMinimumSize</enum>
    </property>
    <item>
     <layout class="QHBoxLayout" name="column_hlayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetMinimumSize</enum>
      </property>
      <item>
       <layout class="QVBoxLayout" name="lasers_vlayout">
        <property name="sizeConstraint">
         <enum>QLayout::SetMinimumSize</enum>
        </property>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Maximum</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="PyDMDrawingLine" name="PyDMDrawingLine">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>20</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="toolTip">
         <string/>
        </property>
        <property name="rotation" stdset="0">
         <double>90.000000000000000</double>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="xpm_vlayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
            <underline>false</underline>
           </font>
          </property>
          <property name="text">
           <string>SC Pattern Metadata</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="1">
           <widget class="PyDMLabel" name="pattern_name_rbv">
            <property name="toolTip">
             <string/>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="PyDMLabel" name="time_source_rbv">
            <property name="toolTip">
             <string/>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="offset_rbv_label">
            <property name="text">
             <string>Offset RBV</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="time_slot_label">
            <property name="text">
             <string>Time Slot</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="name_label">
            <property name="text">
             <string>Name</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="PyDMLabel" name="time_slot_rbv">
            <property name="toolTip">
             <string/>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="time_source_label">
            <property name="text">
             <string>Time Source</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="rate_rbv_label">
            <property name="text">
             <string>Rate RBV</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="PyDMLabel" name="rate_rbv">
            <property name="toolTip">
             <string/>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="PyDMLabel" name="offset_rbv">
            <property name="toolTip">
             <string/>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="time_slot_mask_label">
            <property name="text">
             <string>Time Slot Mask</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="PyDMLabel" name="time_slot_mask_rbv">
            <property name="toolTip">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="PyDMLabel" name="xpm_label">
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="toolTip">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="PyDMNTTable" name="xpm">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>470</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="toolTip">
           <string/>
          </property>
          <property name="alarmSensitiveContent" stdset="0">
           <bool>false</bool>
          </property>
          <property name="alarmSensitiveBorder" stdset="0">
           <bool>true</bool>
          </property>
          <property name="PyDMToolTip" stdset="0">
           <string/>
          </property>
          <property name="monitorDisp" stdset="0">
           <bool>false</bool>
          </property>
          <property name="channel" stdset="0">
           <string/>
          </property>
          <property name="readOnly" stdset="0">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::MinimumExpanding</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="PyDMDrawingLine" name="PyDMDrawingLine_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>1280</width>
     <height>16</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="MinimumExpanding" vsizetype="Minimum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="toolTip">
    <string/>
   </property>
  </widget>
  <widget class="PyDMLabel" name="screen_title">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>1151</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="toolTip">
    <string/>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PyDMNTTable</class>
   <extends>QWidget</extends>
   <header>pydm.widgets.nt_table</header>
  </customwidget>
  <customwidget>
   <class>PyDMLabel</class>
   <extends>QLabel</extends>
   <header>pydm.widgets.label</header>
  </customwidget>
  <customwidget>
   <class>PyDMDrawingLine</class>
   <extends>QWidget</extends>
   <header>pydm.widgets.drawing</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
